operations/preinstall/Preinstall.md

```diff
@@ -0,0 +1,22 @@
+## Pre-Install Steps
+
+This section provides information on what needs to be done before
+an initial install of CSM.
+
+### River BMC Credentials
+
+It is necessary to set the default credentials of all River BMCs so that
+CSM Hardware Management can interact with Redfish.
+
+This procedure is outlined in [Change River BMC Credentials](Change_River_BMC_Credentials.md).
```

Comment:
Else where in the documentation `River` BMCs have been referenced as air-cooled BMCs, and `Mountain` and `Hill` BMCs have referenced as liquid-cooled BMCs.

Reply:
Done.